server/game/cards/locations/04/towerofthesun.js

```diff
@@ -0,0 +1,20 @@
+const DrawCard = require('../../../drawcard.js');
+
+class TowerOfTheSun extends DrawCard {
+    setupCardAbilities(ability) {
+        this.reaction({
+            when: {
+                onCardPlayed: (event, player, card) => card.getType() === 'event'
```

Comment:
The `getType()` check here will be redundant once #573 is merged but doesn't hurt anything and is necessary right now. So 👍